website/README.md

```diff
@@ -0,0 +1,193 @@
+This website was created with [Docusaurus](https://docusaurus.io/).
```

Comment:
This doc should be removed or at least edited I think.

Reply:
removed.